modules/default-requirements-central-services.adoc

```diff
@@ -50,6 +63,8 @@ The following table lists the minimum memory and storage values required to inst
 | 100 GiB
 |===
 
+Central requires Central-DB to store data. The following table lists the minimum memory and storage values required to install and run Central-DB.
+
 |===
 | Central DB | CPU | Memory | Storage
```

Comment:
I have changed it to `Central DB` in the rest of the document. We are using `Central DB` in other documents and it would be better to stay consistent.